src/pages/LEVANTE/RegisterUsers.vue

```diff
@@ -155,6 +155,35 @@ const allFields = [
     dataType: 'string',
   },
 ];
+// Used to remove all other fields in the uploaded CSV
+const allFieldsIdsList = allFields.map((a) => a.field);
+const sanitizeAndValidateSubmittedUsers = (users) => {
+  const userIdMap = {};
+  const filteredFieldsUsers = users.map((user) => {
+    const acc = {};
+    Object.keys(user).forEach((uKey) => {
+      if (allFieldsIdsList.includes(uKey)) {
+        acc[uKey] = user[uKey];
+      }
+    });
+    // Allowing 0 as a valid id
+    const idList = [user['childId'], user['teacherId'], user['parentId']].filter((a) => 0 || !!a);
+    if (idList.length === 0) {
+      throw new Error(`User Row validation:, No Child, Teacher or Parent Id found for: ${JSON.stringify(user)}`);
```

Comment:
Sorry this is on me, I should have given you more context. We allow the site researchers to register children only. They don't always have the parent/teacher relationships beforehand. In this case child, teacher, and parent id columns would be left blank.